src/components/feed.jsx

```diff
@@ -6,7 +6,8 @@ import firebase from "../firebaseConfig";
 
 class Feed extends Component {
     state = {
-        firstName: ""
+        firstName: "",
+       
```

Comment:
try not to change if u dont do anything w the change. this goes for some of the other code in this PR. Code that changed should only serve functional purposes